sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/models/BlobSetTierHeaders.java

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.storage.blob.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+/**
+ * Defines headers for SetTier operation.
+ */
+@JacksonXmlRootElement(localName = "Blob-SetTier-Headers")
+public class BlobSetTierHeaders {
+    /*
+     * If a client request id header is sent in the request, this header will
+     * be present in the response with the same value.
+     */
+    @JsonProperty(value = "x-ms-client-request-id")
+    private String clientRequestId;
+
+    /*
+     * This header uniquely identifies the request that was made and can be
+     * used for troubleshooting the request.
+     */
+    @JsonProperty(value = "x-ms-request-id")
+    private String requestId;
+
+    /*
+     * Indicates the version of the Blob service used to execute the request.
+     * This header is returned for requests made against version 2009-09-19 and
+     * newer.
+     */
+    @JsonProperty(value = "x-ms-version")
+    private String version;
+
+    /*
+     * The errorCode property.
+     */
+    @JsonProperty(value = "x-ms-error-code")
```

Comment:
Can we get rid of this?

Reply:
Hmm. the getProperties method has errorCode in its headers. You're right though that its odd to return it

Reply:
Maybe on the convenience layer we can remove things such as this, however, if we are going to use AutoRest to generate the service layer then these headers will probably end up there anyways.